cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowToken.java

```diff
@@ -42,7 +42,10 @@
   /**
    * Put the specified key and value into the {@link WorkflowToken}.
    * The token may store additional information about the context in which
-   * this key is being set, for example, the unique name of the workflow node.
+   * this key is being set, for example, the unique name of the workflow node
```

Comment:
I think intention here is to state that we store some more information with every put operation. Node name or the Workflow name is just part of that information. In future we might want to store more contextual information such as put time etc. So saying context is the unique node name wont be correct.